recipes/libcamera/meta.yaml

```
@@ -0,0 +1,99 @@
+{% set libcamera_version = "0.4.0" %}
+{% set libcamera_rpi_version = "0.4.0+rpt20250213" %}
+{% set rpi_url_safe_version = libcamera_rpi_version | urlencode %}
+
+package:
+  name: libcamera
+  version: {{ libcamera_version }}
+
+source:
+{% if variant == "upstream" %}
+  git_url: https://git.libcamera.org/libcamera/libcamera.git
+  git_rev: v{{ libcamera_version }}
+{% elif variant == "rpi_fork" %}
+  url: https://github.com/raspberrypi/libcamera/releases/download/v{{ rpi_url_safe_version }}/libcamera-{{ rpi_url_safe_version }}.tar.xz
+  sha256: 764b9b2ab16dbc4e90cd858cd2019be76e087e093bbdd08048d87d0a704f8e28
+{% endif %}
+  patches:
+    - memfd_symbols.patch
+
+build:
+  number: 0
+  skip: true  # [not linux]
+  string: "py{{ CONDA_PY }}_h{{ PKG_HASH }}_{{ PKG_BUILDNUM }}_{{ variant }}"
+
+requirements:
+  build:
+    - cmake
+    - ninja
+    - meson
+    # Needed for some build processes???
+    - ply
+    - pyyaml
+    - jinja2
+    - pkg-config
+    - {{ compiler('cxx') }}
+    - {{ compiler('c') }}
+    - {{ stdlib('c') }}
+  host:
+    - libgl-devel
+    - python
+    - pybind11
+    - jinja2
+    - libudev
+    - libudev1
+    - gstreamer
+    - libevent
+    - libdrm
+    - libtiff
+    - qt6-main
+    - libdrm
+    - libjpeg-turbo
+    - yaml
+    - libunwind
+    - sdl2
+    - openssl
+    - libboost-devel
+    - pyyaml
+    - ply
+  run:
+    - python
+test:
+  commands:
+    - test -f $PREFIX/lib/libcamera.so
+    - test -f $PREFIX/include/libcamera/libcamera/libcamera.h
+
+outputs:
+  - name: libcamera
+    test:
+      commands:
+        - cam -l
+  - name: libcamera-rpi
+    build:
+      skip: true  # [variant == "upstream"]
+    requirements:
+      run:
+        # I wish we could pin exact... but I think with the python build matrix
+        # It makes things complicated.
+        - {{ pin_subpackage('libcamera') }}
```

```suggestion
        - libcamera {{ version }}=*_{{ PKG_BUILDNUM }}_{{ variant }}
```

this should help it be pinned correctly to the build number and the variant while leaving python floating.

---

I'll wait for the ci to complete before committing, thank you